mesa/__init__.py

```diff
@@ -6,8 +6,7 @@
 
 import datetime
 
-import mesa.space as space
-import mesa.time as time
+from mesa import space, time
```

Comment:
This is the cause of the circular import in the CI fails.

Reply:
Okay